packages/@expo/cli/src/lint/lintAsync.ts

```diff
@@ -0,0 +1,111 @@
+import JsonFile, { type JSONObject } from '@expo/json-file';
+import * as PackageManager from '@expo/package-manager';
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs/promises';
+import path from 'path';
+
+import { Log } from '../log';
+import { selectAsync } from '../utils/prompts';
+
+const WITH_PRETTIER = `module.exports = {
+  root: true,
+  extends: ["expo", "prettier"],
+  plugins: ["expo", "prettier"],
+  rules: {
+    "prettier/prettier": ["warn"],
+  },
+};
+
+`;
+
+const ESLINT_ONLY = `module.exports = {
```

Comment:
Add a type comment annotation if there's a common one for eslint (?)

Reply:
I'm not sure there is. Their own [eslint.config.js](https://github.com/eslint/eslint/blob/main/eslint.config.js) doesn't have a jsdoc for the export but they use jsdoc otherwise 🤔